### PR TITLE
Admin homepage get data

### PR DESCRIPTION
**Pull request by annq3012**
## Description
Get Count User, Book, Category
## Reference Requirements
- Product Backlog ID: PBI-1
- Task ID: TI-1

## Reference Isuees
#{issue_id}

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:
-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)
- select count(*) as aggregate from `users` where `users`.`deleted_at` is null;
- select count(*) as aggregate from `books`;
- select count(*) as aggregate from `categories`;

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)
category
book
user

### CODE REVIEW

**Comment by at-anhngo** [comment]
LGTM

**Review by ghost — APPROVED** [review]

File: app/Helpers/common.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  /**
+   * Get percent progress name of database
+   *
+   * @param string $name name of database
+   *
+   * @return float
+   */
+    function getCount($name)
+    {
+        $count = '';
```
Comment (at-duyho) [inline app/Helpers/common.php]: - you should define clearly type for count variable.

File: app/Helpers/common.php [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        switch ($name) {
+            case 'users':
+                $count = User::count();
+                break;
+            case 'books':
+                $count = Book::count();
+                break;
+            default:
+                $count = Category::count();
+                break;
```
Comment (at-duyho) [inline app/Helpers/common.php]: - when you use block default, you should not need break statement.

File: app/Helpers/common.php [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+   *
+   * @param string $name name of database
+   *
+   * @return float
+   */
+    function getCount($name)
+    {
+        $count = '';
+        switch ($name) {
+            case 'users':
```
Comment (at-duyho) [inline app/Helpers/common.php]: - you should define constant. I think you should use: nameClass::class.

File: app/Helpers/common.php [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+if (!function_exists('getCount')) {
+  
+  /**
+   * Get percent progress name of database
+   *
+   * @param string $name name of database
+   *
+   * @return float
+   */
+    function getCount($name)
```
Comment (at-duyho) [inline app/Helpers/common.php]: - what's purpose?

**Review by ghost — CHANGES_REQUESTED** [review]
check my comment. thank!

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**